src/Illuminate/Auth/Console/stubs/make/tests/AuthTest.stub

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use {{namespace}}User;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class AuthTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected $name = 'Your name';
+    protected $email = 'a@valid.email';
+    protected $password = 'secret';
+
+    public function test_registration()
+    {
+        $this->visit('register')
+            ->type($this->name, 'name')
+            ->type($this->email, 'email')
+            ->type($this->password, 'password')
+            ->type($this->password, 'password_confirmation')
+            ->press('Register')
+            ->seeInDatabase('users', [
+                'name'  => $this->name,
+                'email' => $this->email,
+            ])
+            ->see('Welcome')
+            ->seePageIs('/')
+            ->seeIsAuthenticated();
```

Comment:
Should add `$this->assertTrue(Hash::check($this->password, User::first()->password));`